ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryRoutingFunctionalTest.java

```diff
@@ -360,11 +343,10 @@ public void shouldQueryStandbyWhenActiveDeadStandbyAliveQueryIssuedToRouter() th
   @Test
```

Comment:
I am confused by the title and description of the PR. You say you add a new test case for restoring the active after it restarts but that's not what's happening here: the active gets killed and we query the standby that has lag. Am I missing something?

Reply:
Yes.  There is no standby here.  There's just one server.  It gets up to date with its state store, it gets killed, has it's state store wiped out, and then restarts.  Then that same server is queried.

Reply:
Sorry, this file hasn't been changed materially.  The new file has a new test that does what I described.